### PR TITLE
Breadcrumbs

### PR DESCRIPTION
**Pull request by ptbrowne**
Breadcrumbs from Banks.

I suppose they are different in Drive, maybe we'll have a variant and then
try to reconciliate the two ?

Visible here : https://ptbrowne.github.io/cozy-ui/react/#/Breadcrumbs

### CODE REVIEW

**Comment by y-lohse** [comment]
> I suppose they are different in Drive, maybe we'll have a variant and then try to reconciliate the two ?

They are definitely different. I don't mind adding this version to UI (haven't looked at the code yet), but it does mean that we might have a breaking change on this component later on.

**Comment by ptbrowne** [comment]
Yes if the API changes, we'll have a breaking change.

File: react/Breadcrumbs/index.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          isCurrent
+        />
+      </div>
+    </div>
+  )
+}
+
+Breadcrumbs.propTypes = {
+  className: PropTypes.string,
+  color: PropTypes.oneOf(['default', 'primary']),
```
Comment (y-lohse) [inline react/Breadcrumbs/index.jsx:92]: I suggest renaming this to `theme` to be more in line with other UI components. Color is usually fo passing an actual color value.

File: react/Breadcrumbs/index.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          }
+          className={cx(styles.Breadcrumb__previousButton, styles[color])}
+          onClick={lastPreviousItem.onClick}
+        />
+      )}
+      <div className={cx(styles.Breadcrumb__items, styles[color])}>
+        <div className={styles.Breadcrumb__previousItems}>
+          {previousItems.map(({ name, onClick, tag }, index) => (
+            <BreadcrumbItem
+              key={name}
```
Comment (y-lohse) [inline react/Breadcrumbs/index.jsx:72]:
```suggestion
              key={index}
```

I think `name` might not be unique enough, for example in a folder architecture like `Jobs/Cozy/Cozy/Whatever` we would have identical keys and i'm not sure which one react would keep if only one is removed.

**Comment by ptbrowne** [comment]
Thanks, I made the fixes. I also refactored a bit so that `theme` is only used in the stylus file. Normally the Breadcrumb should not care about the theme. Since it only deals with Text and Icons, they should be automatically take their color from the context.

**Pull request merged**